### DISCUSSION TITLE
main tags - edit

### DISCUSSION THREAD

**Discussion by kalle07** [starik222/BooruDatasetTagManager#114] [Ideas]
can you add the possibility that i can edit the main tags on the right side and there where the tag is in the image-tag there it would be changed.
simple example: i wrote in ever third image a tag "close-up" but i want change it to "closeup", so it would be simple to do it in the main tag

;)

**Comment by starik222** [comment]
What don't you like about the "Replace tag" function?

**Comment by kalle07** [comment]
i see ... omg yes

than maybe an option to choose some of my pics (mark it) and only for them i can generate a new tag

**Reply by starik222** [reply]
Select these images via ctrl and add the tag as you would for a single image.

**Reply by kalle07** [reply]
oh, THX !
